src/metrics/kms-service.ts

```diff
@@ -44,7 +44,12 @@ export class KMSService {
 
   // Node 18 or AWS SDK V3
   public async decryptV3(buffer: Buffer): Promise<string> {
-    const { KMSClient, DecryptCommand } = require("@aws-sdk/client-kms");
+    let KMSClient, DecryptCommand
+    try {
```

Comment:
Yeah, I'm going to link to this PR as documentation for the full explanation, as it's relevant to any external contributor as well as internal (so it can't just go into our internal-only wiki and doesn't really belong in the readme)

Reply:
I added comments on both branches because presumably at some point we'll delete the aws-sdk@v2 branch.

Reply:
Makes sense, thanks a lot!